app/src/main/java/de/westnordost/streetcomplete/quests/LeaveNoteDialog.java

```diff
@@ -59,6 +61,20 @@ public void onClick(View v)
 		return view;
 	}
 
+	@Override public void onViewCreated(View view, @Nullable Bundle savedInstanceState)
+	{
+		super.onViewCreated(view, savedInstanceState);
+		if(savedInstanceState == null)
+		{
+			getChildFragmentManager().beginTransaction().add(R.id.attachPhotoFragment, new AttachPhotoFragment()).commit();
+		}
```

Comment:
By the way, @ENT8R, what I told you about nested fragments like I did it for MapFragment + MapControlsFragment turns out to be wrong. The Android documentation explicitly states that it is not possible to have nested fragments in layout xmls. Instead, they have to be added programmatically always.

This is only no problem currently with the MapControlsFragment because it never gets removed and added again. Perhaps I should change this for the MapControlsFragment as well.